app/models/main.py

```diff
@@ -1479,6 +1479,16 @@ def serialize(self, with_users=False, with_clarification_questions=False):
                 'withdrawnAt': self.withdrawn_at.strftime(DATETIME_FORMAT)
             })
 
+        if self.unsuccessful_at:
+            data.update({
+                'unsuccessfulAt': self.unsuccessful_at.strftime(DATETIME_FORMAT)
+            })
+
+        if self.cancelled_at:
+            data.update({
+                'cancelledAt': self.cancelled_at.strftime(DATETIME_FORMAT)
+            })
+
```

Comment:
Good catch on these! I honestly thought they had already been included. I'll have a quick check on the Briefs FE to see if we were expecting them in the response already...

Reply:
Confirmed that Briefs FE is only checking the status, not the datestamp.